api/server/router/volume/volume.go

```diff
@@ -30,6 +32,7 @@ func (r *volumeRouter) initRoutes() {
 		// POST
 		router.NewPostRoute("/volumes/create", r.postVolumesCreate),
 		router.NewPostRoute("/volumes/prune", r.postVolumesPrune),
+		router.NewPostRoute("/volumes/{name:.*}/update", r.postVolumesUpdate),
```

Comment:
I know we made that mistake for some other endpoints, but wondering if we can make this `"/volumes/{name:.*}"` (so without the trailing `/update`). The endpoint would then be `POST`, or if we want to do it "correct"; `PUT` (if we require the whole definition to be sent) or `PATCH` (if we accept "partial" definitions).

Removing the `/update` removes potential confusing URLs, for example, a volume named `foo/update` would otherwise be accessed as `/volumes/foo/update/update`

Reply:
I won't argue this at all, pleased to do it the right way for volumes.

Reply:
as an uninvolved busybody ... this still  looks like "create" and "prune" will be invalid volume names?

Reply:
Arf, you're right; we can't change specifically due to those existing, so we must have the suffix 🤦‍♂️

My bad; didn't think it through properly

Reply:
the routes are:
`GET /volumes`
`GET /volumes/{name:.*}`
`POST /volumes/create`
`POST /volumes/prune`
`POST /volumes/{name:.*}/update`
`DELETE /volumes/{name:.*}`

Replace
`POST /volumes/{name:.*}/update`
with 
`PUT /volumes/{name:.*}`
for the update operation and there is no conflict.

Reply:
Yes, `PUT` (or `PATCH`) works for me